### PR TITLE
Please Review - Completed

### PR DESCRIPTION
**Pull request by firoz1905**

### CODE REVIEW

File: Number of Islands [hunk 1]
```diff
@@ -1,6 +1,56 @@
...
+                    count+=1
+                    self.helper(grid,i,j)
+        return count
+                    
+    def helper (self,grid,i,j):
+            # Base Case
+            if (i<0 or j<0 or i>=self.m or j>=self.n or grid[i][j]=='0'):return
+                
+            # logic
+            grid[i][j]=='0'
```
Comment (Sumesh-H) [inline Number of Islands]: grid[i][j] = '0'

**Review by Sumesh-H — COMMENTED** [review]
Good work, Firoz. Review inline comment.

**Comment by Sumesh-H** [comment]
Good work, Firoz. Review inline comment.

File: Number of Islands (hunk 1 above)
Comment (firoz1905) [reply Number of Islands]: Thank you !. Updated

**Pull request closed without merging**